src/creatures/combat/spells.cpp

```diff
@@ -630,6 +630,38 @@ void Spell::setWheelOfDestinyBoost(WheelSpellBoost_t boost, WheelSpellGrade_t gr
 	}
 }
 
+void Spell::getCombatDataAugment(std::shared_ptr<Player> player, CombatDamage &damage) {
+	if (!(damage.instantSpellName).empty()) {
+		std::vector<std::shared_ptr<Item>> equippedAugmentItems = player->getEquippedAugmentItems();
+		for (std::shared_ptr<Item> item : equippedAugmentItems) {
+			Augments augments = item->getAugmentsBySpellName(damage.instantSpellName);
+			for (auto &augment : augments) {
+				damage.lifeLeech += augment->type == AUGMENT_LIFELEECHAMOUNT ? augment->value : 0;
+				damage.manaLeech += augment->type == AUGMENT_MANALEECHAMOUNT ? augment->value : 0;
+				damage.criticalDamage += augment->type == AUGMENT_CRITICALHITDAMAGE ? augment->value : 0;
+				if (augment->type == AUGMENT_INCREASED_DAMAGE || augment->type == AUGMENT_POWERFUL_IMPACT || augment->type == AUGMENT_STRONG_IMPACT) {
+					const float augmentPercent = augment->value / 10000.0;
```

Comment:
I think you can adjust your calculation here.

Reply:
I'll reduce to percentage and adjust the calculation with the percentage.